client/dashboard/sites/overview-latest-activity-card/index.tsx

```diff
@@ -11,11 +12,17 @@ import TimeSince from '../../components/time-since';
 import { isSelfHostedJetpackConnected } from '../../utils/site-types';
 import type { ActivityLogEntry, Site } from '../../data/types';
 import type { Field } from '@wordpress/dataviews';
+import './style.scss';
 
 const fields: Field< ActivityLogEntry >[] = [
 	{
 		id: 'gridicon',
-		render: () => null, // TODO: render the gridicon
+		render: ( { item } ) => (
+			<Gridicon
+				icon={ item.gridicon }
```

Comment:
Are we limited to gridirons here because that’s what’s returned by the API?

Ideally we should be using icons from WPDS everywhere in the hosting dashboard. Activity log should be updated to use them too.

Reply:
Yes, the API literally returns a field called `gridicon` which is the ID of a GridIcon 🥹 let me see if it's feasible to create a mapping from it to WPDS...

Reply:
@matt-west I updated the PR to use WP icons. It's a best-effort mapping from all gridicons used in the activity log, to suitable icons in WPDS Icons.